util/network-devp2p/src/host.rs

```diff
@@ -813,7 +808,7 @@ impl Host {
 			}
 
 			if kill {
-				self.kill_connection(token, io, true);
+				self.kill_connection(token, io);
```

Comment:
Should we `note_failure`?

Reply:
in some cases of `kill = true`, we already noted failure, when in others we did not. I'll fix it.